src/lib.rs

```diff
@@ -680,6 +681,71 @@ impl ConfigFile {
     self.json.unstable.iter().any(|v| v == name)
   }
 
+  pub fn to_exports_config(
+    &self,
+  ) -> Result<IndexMap<String, String>, AnyError> {
```

Comment:
Oh, I was just thinking that it would probably be good to have a way to get an export resolved to a url. I'll have this return a struct that wraps this hashmap in order to make that easier.